### ISSUE TITLE
Clarifying the distinction between URI expansion and PURL

### ISSUE THREAD

**Issue by matentzn** [biopragmatics/bioregistry#666]
It is absolutely critical for data integration to know what _one_ way a CURIE should be expanded back to (in terms of URL).

for example, the `chebi` prefix should _always_ be translated to http://purl.obolibrary.org/obo/CHEBI_ which is what CHEBI.owl is using. 

It seems to me after listening to today's workshop that we should consider distinguishing between:

- PURL
- preferred URI expansion

Maybe these really are different.

**Comment by balhoff** [comment]
I agree—I came here to open an issue but found this existing one. I think Bioregistry should somewhere indicate what the canonical expanded URI form is for an identifier. For example, on the GO page (https://bioregistry.io/registry/go) there are a number of ways listed to turn a GO term CURIE into a URL, but nowhere does it state that the officially endorsed and preferred URI identifier form for a GO term is `http://purl.obolibrary.org/obo/GO_nnnnnnn`.

**Comment by cthoyt** [comment]
So maybe there are two axes we should annotate:

1. Official suggestion from first party
2. This is a PURL

I think it's totally reasonable to keep track of first party annotations. I will be updating the data model in the future also to include a tag if a URI format is appropriate for RDF usage,

Can take responsibility for writing a PR that includes an explanation of what makes something a PURL and what are the curation guidelines for looking at existing URI format strings and determining if they're PURLs? This can go in https://github.com/biopragmatics/bioregistry/blob/main/src/bioregistry/app/templates/meta/summary.html on first try but we could move it later.

**Comment by matentzn** [comment]
I put it on my list to draft a blurp, but I am in a paper jam atm, so not sure how quick I will get to it.
